turbinia/api/cli/turbinia_client/helpers/formatter.py

```diff
@@ -506,7 +506,7 @@ def generate_data_frame(self) -> pandas.DataFrame:
     Returns: 
       data_frame (DataFrame): Statistics table in pandas DataFrame format.
     """
-    for stat_group, stat_dict in self._statistics.items():
+    for stat_group, stat_dict in self._statistics.__dict__.items():
```

Comment:
It seems odd that we need to use ```__dict__``` here considering that self._statistics should already be a dictionary according to the type hint. Is there a reason why this would be something other than a dictionary that we need to call ```__dict__```?

Reply:
Thanks, statistics ended up being a BaseModel class after I changed the API response from JSON to basemodel. But that makes sense, I'll call model_dump before passing to the formatter to pass a dict to that function.